app/pods/classroom/timeline/overview/route.js

```diff
@@ -1,8 +1,9 @@
 import Route from '@ember/routing/route';
 import { hash } from 'rsvp';
+import { inject } from '@ember/service';
 
 export default Route.extend({
-  api:Ember.inject.service('api'),
+  api: inject.service('api'),
```

Comment:
This won't work. Either use `inject('api')` or

Reply:
Done!